### ISSUE TITLE
Typescript check error after upgrade to 2.0.0 or 2.0.1

### ISSUE THREAD

**Issue by pavolgolias** [i18next/i18next-intervalPlural-postProcessor#219]
## 🐛 Bug Report

After upgrade to version 2.0.0 or 2.0.1 of i18next-intervalplural-postprocessor there is an typescript error check in file https://github.com/i18next/i18next-intervalPlural-postProcessor/blob/master/index.d.ts, row 9 -> Top-level declarations in .d.ts files must start with either a 'declare' or 'export' modifier.

## To Reproduce

It is about this code shown below (file index.d.ts). To reproduce this error just install i18next-intervalplural-postprocessor dependecy and make sure you have `"skipLibCheck": false` in your `tsconfig.json` file.

```js
const intervalPlural: PostProcessorModule & {
	name: 'interval';
	type: 'postProcessor';
	options: IntervalPluralOptions;
	setOptions: (options: Partial<IntervalPluralOptions>) => void;
};
```

## Expected behavior

There should be no errors on typescript check.
Maybe this fix below should work (I am not sure if it is enough).
```js
export const intervalPlural: PostProcessorModule & {
	name: 'interval';
	type: 'postProcessor';
	options: IntervalPluralOptions;
	setOptions: (options: Partial<IntervalPluralOptions>) => void;
};
```

## Environment

- *node version*: 12.18.0
- *npm version*: 6.14.5
- *browser*: Chrome v 83.0.4103.116
- *i18next version*: 19.5.4
- *i18next-intervalplural-postprocessor version*: 2.0.1
- *typescript version*: 3.9.6
- *os*: Win 10

**Comment by jamuhl** [comment]
https://github.com/i18next/i18next-intervalPlural-postProcessor/pull/218

@seleb please have a look at this...seems to be introduced with your PR ...

**Comment by pavolgolias** [comment]
@seleb @jamuhl Is there something new? I see merged PR, is it going to be new relese 2.0.2 published?
